universal-application-tool-0.0.1/app/services/program/CsvExportConfig.java

```diff
@@ -0,0 +1,6 @@
+package services.program;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class CsvExportConfig {}
```

Comment:
(blocking): are these placeholders? 

Should they pdf and csv config share a common interface?

Reply:
Yes to placeholders, and no they cannot share a common interface.  This is where things like "this field maps to this result" goes for PDFs, or a list of headers for CSVs, or the base template for a PDF.